lib/testing/src/lib/core/pages/pagination.page.ts

```diff
@@ -41,6 +41,9 @@ export class PaginationPage {
         await browser.executeScript(`document.querySelector('div[class*="adf-pagination__perpage-block"] button').click();`);
         await BrowserVisibility.waitUntilElementIsVisible(this.pageSelectorDropDown);
         const itemsPerPage = element.all(by.cssContainingText('.mat-menu-item', numberOfItem)).first();
+        await BrowserVisibility.waitUntilElementIsPresent(itemsPerPage);
```

Comment:
Doesn't make sense to have all these 3 waits.
Please find a better solution

Reply:
The best working solution I could come up with is having the waitUntilElementIsPresent and got rid of the other 2 as, they are being used in the BrowserActions click method anyway.
Tried adding the same waitUntiElementIsPresent() in the BrowserActions method instead of in the Pagination Page, doesn't work for the tests!